bin/primer_designer_region.py

```diff
@@ -1636,8 +1636,11 @@ def method_blurb(self, args, seqs=None):
             f'Created at {datetime.now().strftime("%d/%m/%Y %H:%M:%S")}'
         )
         lines.append(f'SNP file used: {Path(SNP).stem}')
+
+        ref_genome = Path(REFERENCE).stem
+        ref_genome = ref_genome[:70] if len(ref_genome) > 70 else ref_genome
```

Comment:
here you don't need to check if its too long to slice:
```
>>> s='somestring'
>>> s[:70]
'somestring'
```

but also should we just chop it off if its too long, or drop it to a new line and indent or something?

Reply:
added indent